packages/earth-map/src/pages/profile/component.tsx

```diff
@@ -30,20 +48,48 @@ export function ProfileComponent(props: IProps) {
 
       setIsLoading(false);
     })();
-  }, []);
+  }, [refreshProfile]);
+
+  async function onSubmit(e?, setIsEditing?, setIsLoading?, setServerErrors?) {
```

Comment:
all cards from profile use a different api, please rename onSubmit to something relevant for editing name